sbi/inference/base.py

```diff
@@ -7,7 +7,7 @@
 from copy import deepcopy
 from datetime import datetime
 from pathlib import Path
-from typing import Callable, Dict, List, Optional, Tuple, Union, cast
+from typing import Callable, Dict, List, Optional, Tuple, Union, cast, Any
```

Comment:
Make sure to run `isort`. You can also set up a git hook to do this automatically: https://timothycrosley.github.io/isort/#git-hook